migrations/versions/62308cd181b2_add_overseas_region.py

```diff
@@ -0,0 +1,114 @@
+"""Add Overseas region
+
+Revision ID: 62308cd181b2
+Revises: dimension_position_not_null
+Create Date: 2019-11-08 15:58:57.694684
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+from application.dashboard.view_sql import (
+    drop_all_dashboard_helper_views,
+    latest_published_measure_versions_view,
+    latest_published_measure_versions_by_geography_view,
+    ethnic_groups_by_dimension_view,
+    classifications_by_dimension,
+)
+
+# revision identifiers, used by Alembic.
+revision = "62308cd181b2"
+down_revision = "dimension_position_not_null"
+branch_labels = None
+depends_on = None
+
+upgrade_lowest_level_of_geography = """
+    UPDATE lowest_level_of_geography SET position =  2 WHERE name = 'United Kingdom';
+    UPDATE lowest_level_of_geography SET position =  3 WHERE name = 'Country';
+    UPDATE lowest_level_of_geography SET position =  4 WHERE name = 'Region';
+    UPDATE lowest_level_of_geography SET position =  5 WHERE name = 'Local authority upper';
+    UPDATE lowest_level_of_geography SET position =  6 WHERE name = 'Local authority lower';
+    UPDATE lowest_level_of_geography SET position =  7 WHERE name = 'Police force area';
+    UPDATE lowest_level_of_geography SET position =  8 WHERE name = 'Clinical commissioning group';
+    UPDATE lowest_level_of_geography SET position =  9 WHERE name = 'Fire and rescue service area';
+    INSERT INTO lowest_level_of_geography (name, position) VALUES ('UK and Overseas', 1);
+"""
+
+downgrade_lowest_level_of_geography = """
+    DELETE FROM lowest_level_of_geography WHERE name = 'UK and Overseas';
+    UPDATE lowest_level_of_geography SET position =  1 WHERE name = 'United Kingdom';
+    UPDATE lowest_level_of_geography SET position =  2 WHERE name = 'Country';
+    UPDATE lowest_level_of_geography SET position =  3 WHERE name = 'Region';
+    UPDATE lowest_level_of_geography SET position =  4 WHERE name = 'Local authority upper';
+    UPDATE lowest_level_of_geography SET position =  5 WHERE name = 'Local authority lower';
+    UPDATE lowest_level_of_geography SET position =  6 WHERE name = 'Police force area';
+    UPDATE lowest_level_of_geography SET position =  7 WHERE name = 'Clinical commissioning group';
+    UPDATE lowest_level_of_geography SET position =  8 WHERE name = 'Fire and rescue service area';
+"""
+
+old_enum = ("ENGLAND", "WALES", "SCOTLAND", "NORTHERN_IRELAND", "UK")
+new_enum = old_enum + ("OVERSEAS",)
+
+old_type = sa.Enum(*old_enum, name="uk_country_types")
```

Comment:
I'm pretty sure you can add values to an existing ENUM (but not delete them), so I don't think you should need to create all these new ones and switch round?

Reply:
I did try the `ALTER TYPE` route but it needs a separate connection. I've read a lot of solutions around this and this is preferred route. I also like that it's symmetrical with the downgrade. You have to do the switch around with downgrade anyway.